### PR TITLE
Rewrite Custom Sender Guide, add useful links

### PR DESCRIPTION
**Pull request by marklewin**
## Description

A light rewrite of the existing concept doc and the addition of more relevant links.

## Deploy Notes

Check in sample app. No dependence on building blocks, etc

### CODE REVIEW

File: _documentation/messaging/sms/guides/custom-sender-id.md [hunk 1]
```diff
@@ -4,32 +4,39 @@ title: Custom Sender ID
 
 # Custom Sender ID
 
-The Sender ID is the number or text shown on a handset when it displays a message. You set a custom Sender ID to better represent your brand.
+An SMS Sender ID is an easy way to brand your SMS messages so that the person receiving them can quickly see who it is from. Use a custom Sender ID to better represent your brand and maximise readership and response rates.
```
Comment (mheap) [inline _documentation/messaging/sms/guides/custom-sender-id.md]: You can't respond to a custom Sender ID

**Comment by mheap** [comment]
I think I'm getting confused here. What do we mean by custom sender ID? A Nexmo LVN? A different phone number? An alphanumeric sender?

**Comment by marklewin** [comment]
@mheap My understanding is that a custom Sender ID is just alphanumeric text that replaces the phone number in the `from` field of the request.

**Comment by mheap** [comment]
For completeness, @marklewin and I spoke about this. A sender ID can be more than just alphanumeric. The phrase `Custom Sender ID` doesn't really mean anything, as it could be:

* A Nexmo LVN
* Any phone number (which can be replied to, not necessarily via Nexmo e.g. your personal phone number)
* An Alphanumeric sender (which can't be replied to)

**Comment by marklewin** [comment]
I've just committed a new version of the guide (`33bdee7`) that tries to emphasise that `SenderID` isn't a thing in its own right - it's just the contents of the `from` parameter in the request.

**Pull request merged**